docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -168,6 +168,89 @@ public class CounterBean {
 
 == Non-standard Features
 
+=== Eager Instantiation of Beans
+
+[[lazy_by_default]]
+==== Lazy By Default
+
+By default, CDI beans are created lazily, when needed.
+What exactly means "needed" depends on the scope of a bean.
+
+* A *normal scoped bean* (`@ApplicationScoped`, `@RequestScoped`, etc.) is needed when a method is invoked upon an injected instance (contextual reference per the specifitation). 
```

Comment:
```suggestion
* A *normal scoped bean* (`@ApplicationScoped`, `@RequestScoped`, etc.) is needed when a method is invoked upon an injected instance (contextual reference per the specification). 
```

Reply:
This one was fixed while I was reviewing :).

Reply:
I am learning from the master reviewer :wink: